src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/transaction_list.html

```diff
@@ -43,7 +43,12 @@
                             </button>
                         </div>
                     {% elif trans.state == 'error' or trans.state == 'already' %}
+                        <input type="text" class="form-control" placeholder="{% trans "Order code" %}">
```

Comment:
Hmm, this also adds the option in ``trans.state == 'error'``, do we really want that?

Reply:
Good point - changed that.